js/templates/userPage/userPage.html

```diff
@@ -0,0 +1,58 @@
+<nav id="pageTabBar" class="barLg clrP clrBr">
+
+      <div class="flexVCent pageTabs">
+        <!--consider turning tabs into a reusable view-->
+        <div>
+          <div class="userIcon disc clrBr2 clrSh1" style="background-image: url('../imgs/defaultUser.png')"></div>
+        </div>
+        <div>
+          <h1 class="h2 txUnb txUnl"><%= ob.name %></h1>
+          <div class="txt5b">
+            <span>Stars(x)</span>
+            <span>
+              <% if (ob.followsYou) { %>
+                <%= ob.polyT('userPage.followsYou') %>
+              <% } %>
+            </span>
+          </div>
+        </div>
+        <div class="flexExpand">
+          <div class="flexHRight flexVCent gutterH clrT2">
+            <a class="btn tab clrBr js-tab" data-tab="Home">Home</a>
+            <a class="btn tab clrBr js-tab" data-tab="Store">Store</a>
+            <a class="btn tab clrBr js-tab" data-tab="Following">Following</a>
+            <a class="btn tab clrBr js-tab" data-tab="Followers">Followers</a>
+          </div>
+        </div>
+      </div>
+
+</nav>
+<div class="banner" <% if(ob.banner){ %>
```

Comment:
The banner has a fixed position behind the rest of the page. This lets it be positioned under the chat sidebar. 

It does mean the banner image doesn't scroll with the rest of the page. @morebrownies how do you feel about that?